gcloud-java-bigquery/src/main/java/com/google/gcloud/bigquery/QueryRequest.java

```diff
@@ -27,7 +27,34 @@
  * Google Cloud BigQuery Query Request. This class can be used to run a BigQuery SQL query and
  * return results if the query completes within a specified timeout. The query results are saved to
  * a temporary table that is deleted approximately 24 hours after the query is run. The query is run
- * through a BigQuery Job whose identity can be accessed via {@link QueryResponse#jobId()}.
+ * through a BigQuery Job whose identity can be accessed via {@link QueryResponse#jobId()}. If the
+ * query does not complete within the provided {@link Builder#maxWaitTime(Long)} the response
+ * returned by {@link BigQuery#query(QueryRequest)} will have {@link QueryResponse#jobComplete()}
+ * set to {@code false} and {@link QueryResponse#result()} set to {@code null}. To obtain query
+ * results you can use {@link BigQuery#getQueryResults(JobId, BigQuery.QueryResultsOption...)} until
+ * {@link QueryResponse#jobComplete()} returns {@code true}.
+ *
+ * <p>Example usage of a query request:
+ * <pre>    {@code
+ *    QueryRequest request = QueryRequest.builder("SELECT field FROM table")
```

Comment:
In this line and the line below, it might be helpful to add an inline comment explaining that the user has to replace "field", "table" and "dataset" with their own values.